corehq/apps/settings/tests/test_forms.py

```diff
@@ -1,32 +1,54 @@
+from unittest.mock import patch
+
 from django.test import SimpleTestCase, override_settings
+
+from two_factor.plugins.registry import registry
+
+from corehq.apps.hqwebapp.two_factor_methods import (
+    HQGeneratorMethod,
+    HQPhoneCallMethod,
+    HQSMSMethod,
+)
+
 from ..forms import HQTwoFactorMethodForm
 
 
 @override_settings(TWO_FACTOR_CALL_GATEWAY=True, TWO_FACTOR_SMS_GATEWAY=True)
 class TestHQTwoFactorMethodForm(SimpleTestCase):
+
     def test_when_phone_support_is_enabled_all_options_are_shown(self):
-        form = HQTwoFactorMethodForm(allow_phone_2fa=True)
+        self.mock_get_methods.return_value = [HQGeneratorMethod(), HQPhoneCallMethod(), HQSMSMethod()]
```

Comment:
Would it be possible to access the registry directly in this test, rather than mock out the method return values? Mocking out the values seems to change the meaning of this test. Prior, it verified that the form with the given constructor arguments would show the expected options. With the options mocked, it doesn't feel safe to make that guarantee anymore

Reply:
Apologies for the late response but given the changes I've made this no longer applies and the test has been reverted to its previous state. Thanks for flagging.